.gitignore

```diff
@@ -352,3 +352,4 @@ Sessionx.vim
 
 imports.prof
 ci-tests
+examples/maskrcnn-example/config.ini
```

Comment:
I think that this should go to a separate `.gitignore` file in the `maskrcnn-example` directory.

Reply:
removed this from main .gitignore, created a new one for the masrcnn-example, added wandb folder and config.ini file.